include/gDPLoadTextureBlock_Runtime.inc.c

```diff
@@ -0,0 +1,67 @@
+#ifndef GDP_LOADTEXTUREBLOCK_RUNTIME_C
+#define GDP_LOADTEXTUREBLOCK_RUNTIME_C
+
+#include "global.h"
+#include "gfx.h"
+
+// The Following arrays must be defined as const in z_lmap_mark.c to appear in rodata
+
+#ifndef GDP_LOADTEXTUREBLOCK_RUNTIME_CONST
+#define ZCONST
+#else
+#define ZCONST const
+#endif
```

Comment:
what is the Z here supposed to mean?

Reply:
Nothing really, It's just to avoid having an identifier named `CONST`

Reply:
well it can probably be more descriptive then. COND_CONST or something to make it clear that its conditional based on something else being defined, or something